composer.json

```diff
@@ -68,6 +68,7 @@
     "rollbar/rollbar-laravel": "^7.0",
     "spatie/laravel-backup": "^6.16",
     "tecnickcom/tc-lib-barcode": "^1.15",
+    "tecnickcom/tcpdf": "^6.5.0",
```

Comment:
Is there a reason we need to add a second PDF library here? IIRC, we already include one:

https://github.com/snipe/snipe-it/blob/master/composer.json#L30

Reply:
The included one (dompdf) requires building from HTML DOM which defeats the purpose of this engine.
TCPDF seems to be the most up-to-date library that lets you build from scratch. Also, tc-lib-barcode from the same author is already included and integrates nicely.

Reply:
> which defeats the purpose of this engine.

given previous comments in this PR it seems to me like the purpose of the engine (or at least using PDF) is to make it easier to align labels for users with sheet-style labels instead of the single label printers. Is that right? 
I think this was discussed a bit in the non-review comments, but it seems like the extra PDF step is cumbersome for users who  have existing workflows that are using single-label printers.

What issues were you having with the existing HTML label layouts that prompted you to make this? would other solutions such as CSS (or even print specific stylesheets), SVG labels, or something else that's closer to HTML help eliminate the need for a new PDF library and make things both 1) as to use as the current method for single label people, and 2) consistently lay out labels in sheets for people who print labels on sheets?

is it possible to craft an HTML dom input to dompdf that causes it to generate PDF's with a certain layout? then it would simplify the problem from "write HTML that perfectly lays out a sheet of labels on all browsers/devices/print methods" to "write HTML that perfectly lays out a sheet of labels specifically for dompdf". Then TCPDF could be eliminated and also the PDFs could be used only by people printing sheets

Reply:
The purpose is to make it easier for sheet-style, tape-style, and single label templates to be repeatable across browsers and devices.

I started out attempting to use 12mm and 24mm PTouch tapes for asset tags, but found it impossible to print tags in a way that fit correctly. Each browser would decide to add extra margins, or flip labels 90 degrees when batch printing, or not trigger auto-cutting for each page.

Then I tried setting up for Avery label sheets instead. Got it to fit my sheet after about an hour of converting mm from a ruler to hundredths of inches and trial-and-error. Then I realised after all that, it's kinda ugly (no offense to the default setup).

Thought "if only there was a format for printing instead of display". Hence PDF. Add support for multiple layouts and fully customisable fields to show and bam, got a stupidly large chunk of code.